ios/file_picker.podspec

```diff
@@ -2,20 +2,35 @@
 # To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
 #
 Pod::Spec.new do |s|
-  s.name             = 'file_picker'
-  s.version          = '0.0.1'
-  s.summary          = 'A new flutter plugin project.'
-  s.description      = <<-DESC
-A new flutter plugin project.
+  s.name                  = 'file_picker'
+  s.version               = '0.0.1'
+  s.summary               = 'A flutter plugin to show native file picker dialogs'
+  s.description           = <<-DESC
+A flutter plugin to show native file picker dialogs.
                        DESC
-  s.homepage         = 'http://example.com'
-  s.license          = { :file => '../LICENSE' }
-  s.author           = { 'Your Company' => 'email@example.com' }
-  s.source           = { :path => '.' }
-  s.source_files = 'Classes/**/*'
-  s.public_header_files = 'Classes/**/*.h'
-  s.dependency 'Flutter'
-  s.dependency 'DKImagePickerController/PhotoGallery'
+  s.homepage              = 'https://github.com/yangyxd/flutter_picker'
```

Comment:
Hi, the author and page should be pointed to the owner of the plugin/package or else leave it as it was. One doesn't become the author just because added a feature/fix.

Reply:
Apologies, that was definitely my intention. Unfortunately I googled the wrong package when I filled this in.